packages/@react-spectrum/table/stories/Table.stories.tsx

```diff
@@ -1368,8 +1369,59 @@ storiesOf('TableView', module)
       </div>
     ),
     {description: {data: 'Using browser zoom should not trigger an infinite resizing loop. CMD+"+" to zoom in and CMD+"-" to zoom out.'}}
+  )
+  .add(
+    'allowsResizing, controlled, no widths',
+    () => (
+      <ControllingResize width={900} />
+    ),
+  {description: {data: `
+    You can use the buttons to save and restore the column widths. When restoring,
+    you will see a quick flash because the entire table is re-rendered. This
+    mimics what would happen if an app reloaded the whole page and restored a saved
+    column width state.
+    `}}
+  )
+  .add(
+    'allowsResizing, controlled, some widths',
+    () => (
+      <ControllingResize width={900} columns={columnsSomeFR} />
+    ),
+  {description: {data: `
+    You can use the buttons to save and restore the column widths. When restoring,
+    you will see a quick flash because the entire table is re-rendered. This
+    mimics what would happen if an app reloaded the whole page and restored a saved
+    column width state.
+    `}}
+  )
```

Comment:
Something weird I noticed:
1. Go to https://reactspectrum.blob.core.windows.net/reactspectrum/ca8f4230c5efb9f248f0a028257ac60093706f78/storybook/index.html?path=/story/tableview--allowsresizing-controlled-some-widths&providerSwitcher-express=false&providerSwitcher-toastPosition=bottom
2. Make the height column wider and save the column widths
3. Restore the column widths via "restore col". Note that it resets the widths back to what they looked like initially in step 1

This also happens in the "allowsResizing, controlled, no widths" story

Reply:
I'll add that to the description. This is a 'some widths' controlled story, so it won't restore widths of columns it doesn't manage